daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -384,6 +384,24 @@ sealed abstract class HasTxNodes[Nid, +Cid] {
       case (acc, _) => acc
     } -- localContracts.keySet
 
+  /** Return all the contract keys created, exercised, fetched or lookup by this transation.
```

Comment:
Can we be a bit more precise here? What exactly does `all` mean? Does it include transient contracts, created and archived within this transaction? That probably then also defines what we do with rollback nodes.

Reply:
done.